blueocean-dashboard/src/main/less/core.less

```diff
@@ -16,7 +16,7 @@
     bottom: auto;
 }
 
-.btn-show-more {
+button.btn-show-more {
     display: table;
```

Comment:
I know you didn't do this @NicuPascu but could you try this with flexbox props on the button? Maybe `align-self:center`? The main container should be `display:flex` already. If it's not, don't worry about it, no need to rejig the whole layout or anything.